www/codeparameter/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "buildbot-codeparameter",
```

Comment:
Won't this increase possibility of accidental name clashes? E.g. what happens if there's an indirect dependency in node_modules of the same name? I think it's worth to prefix our plugins with Buildbot even if they aren't published to npm.

Reply:
The reason I changed it is that it should match the name of the plugin in buildbot (as per legacy way of identifying custom_templates).
I can use another identifier.